.github/workflows/deploy-test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     environment:
       name: test
     env:
-      CLOUDFRONT_ID: "${{ secrets.CLOUDFRONT_ID }}"
+      CLOUDFRONT_ID: '${{ secrets.CLOUDFRONT_ID }}'
```

Comment:
Can you check why these are changing? Ideally then should not.

Reply:
`yarn format:write` changed it because `.prettierrc` enforces single quotes. Is it different if you run it on your env?

Reply:
#177 PR check also failed with the format check error caused by these double quotes.